geth/common/cipher/decrypt.go

```diff
@@ -0,0 +1,40 @@
+package cipher
```

Comment:
Not sure it should be in `geth/common`. We have also `helpers` package where we could put `cipher` into.